drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/ScenarioRunnerServiceImpl.java

```diff
@@ -80,7 +80,7 @@ public SimulationRunResult runTest(final String identifier,
 
         final List<Failure> failureDetails = new ArrayList<>();
 
-        final Result result = runWithJunit(scenarioRunner, failures, failureDetails);
+        final Result result = runWithJunit(path, scenarioRunner, failures, failureDetails);
```

Comment:
@Rikkola 
Related to my other comments on the PR, I have the impression (correct me if I am wrong) that *Path* is used only to forward its internal String representation. Just this simple usage creates an unbreakable bound between at least three repos (directly) and who can tell how many other (indirectly).

Reply:
It needs to be the actual Path, since that is then used in the UI. The user clicks the Path and the asset opens.